Dockerfile

```
@@ -36,5 +21,22 @@ RUN yarn
 RUN yarn clean
 RUN yarn qbuild
 
+# Set up additional UMA packages installed in this docker container.
+# Configuer the across v2 relayer as a "across-relayer" base package.
+WORKDIR /across-relayer
+
+# Clode the relayer code and copy it to the across-relayer directory. Remove the package directory.
+RUN git clone https://github.com/across-protocol/relayer-v2.git
```

```suggestion
RUN git clone https://github.com/across-protocol/relayer-v2.git /across-relayer/relayer-v2
```

Adding the path, you can delete the mv logic